.github/workflows/ci.yml

```diff
@@ -1,8 +1,19 @@
 ---
 name: "CI"
 on:  # yamllint disable
-  - "push"
-  - "pull_request"
+  push:
+  pull_request:
+    paths-ignore:
+      - "nautobot/docs/**"
+      - "examples/**"
+      # The example plugin should probably trigger CI
+      - "!examples/example_plugin"
```

Comment:
In `develop` this still needs to be `examples/dummy_plugin`, since #1235 is only in `next`, and we will need to make sure it's updated to `examples/example_plugin` when we sync this up into `next`.